syntax = "proto3";

message EducationOrderParent {
  string first_name = 1;
  string last_name = 2;
  string email = 3;
  string phone_number = 4;
  int32 school_id = 5;
}

message EducationOrderStudent {
  string first_name = 1;
  string last_name = 2;
  string email = 3;
  string phone_number = 4;
  int32 school_id = 5;
}

message EducationOrderRequest{
  repeated EducationOrderParent parent_order = 1;
  repeated EducationOrderStudent student_order = 2;
}

message EducationOrderResponse{
  bool success = 1;
}

service OrderService{
  rpc EducationOrder(EducationOrderRequest) returns (EducationOrderResponse);
}